client/src/store/new-request-module.ts

```diff
@@ -496,60 +547,64 @@ export class NewRequestModule extends VuexModule {
     },
     {
-      text: 'Move your Business to BC',
+      text: 'Move (continue) into BC a',
       shortDesc: 'Move Request',
       value: 'MVE',
-      blurb: `You have an existing business in another province. You are closing your business there and moving your 
-              business to BC.`
+      blurb: `Create a new business in British Columbia or register a business you formed in another province or 
+              territory, country or with the federal jurisdiction so that you may also conduct business here in BC.`
     },
+    // COMMENTED OUT FOR FUTURE IMPLEMENTATION
+    // {
+    //   text: 'Assume a New Name in BC',
+    //   shortDesc: 'Assumed Name Request',
+    //   value: 'ASSUMED',
+    //   blurb: `You have an existing business in another province. You are closing your business there and moving your
+    //           business to BC, however, the name of your business is already in use in BC`
+    // },
     {
-      text: 'Assume a New Name in BC',
-      shortDesc: 'Assumed Name Request',
-      value: 'ASSUMED',
-      blurb: `You have an existing business in another province. You are closing your business there and moving your 
-              business to BC, however, the name of your business is already in use in BC`
-    },
-    {
-      text: 'Change your Name',
+      text: 'Change the name of an existing',
       shortDesc: 'Change of Name Request',
       value: 'CHG',
-      blurb: `You have an existing business that is registered in BC and you want to change your name. You will need 
-              your incorporation or firm number assigned to you by Registries.`
+      blurb: `You have an existing business that is registered or incorporated in BC and you want to change your name.  
+              You will need your incorporation or firm number assigned to you by BC Registries and Online Services.`
     },
     {
-      text: 'Amalgamate',
+      text: 'Amalgamate two or more',
       shortDesc: 'Amalgamation Request',
       value: 'AML',
-      blurb: 'You are merging with another company and you want a new name.'
+      blurb: 'You have two or more businesses that you want to combine to create a new business.'
     },
     {
-      text: 'Convert to Another Structure',
+      text: 'Change (alter) the business type of a',
       shortDesc: 'Conversion Request',
       value: 'CNV',
-      blurb: `Convert from one business structure to another. Such as converting from a ULC to a BC Corp. You will need 
-              to identify your business with your Corp. #/Firm # (assigned by Registries).`
+      blurb: `You want to alter from one type of corporation to another.  For example you are a limited company and 
+              want to become an unlimited liability company.   You will need the incorporation number assigned to you 
+              by BC Registries and Online Services.`
     },
     {
-      text: 'Restore Using a Historical Name',
+      text: 'Restore or Reinstate a',
```

Comment:
For consistency, should "Reinstate" be un-capitalized?

Reply:
The documentation calls for capitalization here but will follow up since the trend has definitely been away from that pattern.